client/src/components/home/index.tsx

```diff
@@ -93,7 +96,10 @@ function HomePage(props: {
     startTraining,
   } = useWithReviewAnswerState(props.accessToken, props.search);
   const defaultMentor = props.user.defaultMentor._id;
-  const { mentor, isMentorEdited, loadMentor } = useMentor;
+
```

Comment:
We don't need to select the whole active mentor here, I think we just need something like this:

```typescript
  React.useEffect(() => {
    if (!setupStatus || !showSetupAlert) {
      return;
    }
    setShowSetupAlert(!setupStatus.isSetupComplete);
  }, [setupStatus]);
  const mentorId = useActiveMentor(m => m.data?._id || "");
  if (!(mentorId && setupStatus)) {
    return (
      <div>
        <NavBar title="Mentor Studio" mentorId={""} />
        <CircularProgress />
      </div>
    );
  }
  const mentorIsDirty = useActiveMentor(m => m.data.isDirty);
  const mentorSubjectNamesById: Record<string, string> = useActiveMentor(
    m => m.data.subjects.reduce((acc: Record<string, string>, cur: Subject) => {
      acc[cur._id] = cur.name;
      return acc
    }, {})
  );
```